waspc/src/Wasp/Generator.hs

```diff
@@ -54,6 +55,14 @@ genApp spec =
     <++> genDb spec
     <++> genDockerFiles spec
 
+afterWriteChecks :: AppSpec -> Path' Abs (Dir ProjectRootDir) -> IO [GeneratorWarning]
```

Comment:
Maybe we could name it `checkGeneratedCode`? That feels a bit more descriptive on what it does. `afterWriteChecks` tells me the checks are done after the "write", but I am not sure what is being checked -> which is ok if those are super general checks, but I believe they will likely always / for a long time be checks of the generated code, so maybe we can be more precise right now?

Or, we could remove this additional layer / concept to "write checks" and just directly call `DB.warnIfPrismaSchemaChanged` or smth like that.

Not super important though.